lib/create-response-handler.ts

```diff
@@ -120,3 +177,36 @@ export function createResponseHandler (responseScenarios: ResponseScenarios) {
     }
   }
 }
+
+/**
+ * To make handling the passport.authenticate() callback easier we provide a
+ * createIdentityResponseHandler function. This takes separate callbacks for each of the
+ * response scenarios you have to handle and returns a function that will call the
+ * appropriate callback when called by passport.
+ *
+ * NOTE: Should be used by all new services (not using MSA)
```

Comment:
```suggestion
 * Use this function if your service is not using a Matching Service Adapter (MSA).  
```
I'm hoping this is clear enough without adding anything about the VSP...

Reply:
Let's discuss how we can phrase this better. Too many callbacks taking separate callbacks returning functions that will call the right callback 🙈